### PR TITLE
Add: Ubliss Chrome Extension

### PR DESCRIPTION
**Pull request by LordUttam**
Added Main features of new tab extension: Focus, Greeting, Quote, Weather.
View on netlify:
https://ubliss-chrome-extension.netlify.app/

User onboarding:
image

After focus is completed:
image

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
Code looks clean. Really like the folder structure. Good work

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import "./App.css";
+import {
+  Clock,
+  Greet,
+  Focus,
+  UserOnboarding,
+  Quote,
+  Weather,
+} from "./Components/index.js";
```
Comment (NeerajVodala) [inline src/App.js:9]: works even if you stop at "./Components"

File: src/Components/FocusComponents/FocusInput.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+
+function FocusInput(props) {
+  const [focus, setFocus] = useState("");
+  // const setFocusStored = props.focusStoredFunc;
+
+  function storeFocus(e) {
+    if (e.key === "Enter") {
+      localStorage.setItem("focus-of-day", focus);
+      // setFocusStored(focus);
```
Comment (NeerajVodala) [inline src/Components/FocusComponents/FocusInput.jsx]: remainder: remove comments before pushing

File: src/Components/Weather.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  const [weatherIcon, setWeatherIcon] = useState("");
+  const [temperature, setTemperature] = useState("");
+  const [userLocation, setUserLocation] = useState(
+    JSON.parse(localStorage.getItem("user_location"))
+  );
+  const [imgAlt, setImgAlt] = useState("weather icon");
+  const { phaseOfDay } = useTime();
+
+  function getLocation() {
+    // const userLocation = JSON.parse(localStorage.getItem("user_location"));
```
Comment (NeerajVodala) [inline src/Components/Weather.jsx]: remove comment

File: src/Components/Weather.jsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+              );
+            },
+            () => {
+              console.log("Permission denied!");
+            }
+          );
+        });
+      } else {
+        console.log("Geolocation API not available :(");
+      }
```
Comment (NeerajVodala) [inline src/Components/Weather.jsx]: remove console.logs unless you did them on purpose.
you can use console.error instead which makes sense semantically

File: src/Context/time-context.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    17 <= Number(stdTime.slice(0, 2)) &&
+    Number(stdTime.slice(0, 2)) < 20
+  ) {
+    return "evening";
+  } else if (
+    20 <= Number(stdTime.slice(0, 2)) ||
+    Number(stdTime.slice(0, 2)) < 5
+  ) {
+    return "night";
+  }
```
Comment (NeerajVodala) [inline src/Context/time-context.js]: can replace `else if's` with only `if's` as they are returning anyway

**Review by NeerajVodala — COMMENTED** [review]
suggestions:
- use variables for colors and box-shadow [it is more scalable]
- can put index.js file in context folder as well [easier to import] as your are importing them at multiple places

File: src/App.js (hunk 1 above)
Comment (LordUttam) [reply src/App.js:9]: When we are converting to extension, webpack gives errors if file extension is not there.

File: src/Context/time-context.js (hunk 5 above)
Comment (LordUttam) [reply src/Context/time-context.js]: Suggestion taken. :)

**Pull request merged**